src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+/* BOX SIZING - BORDER BOX */
+* {
+  box-sizing: border-box;
+}
+/* EACH LABLE */
+.search {
+  position: relative;
+}
+/* EACH ICON */
+.icon {
+  display: block;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+/* SMALL ICON */
+.icon-small {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+/* EACH INPUT */
+.input {
+  display: block;
+  font-family: "Avenir" , sans-serif;
```

Comment:
we can move it to `html` selector

Reply:
You mean font family?
Inputs doesn't inherit font properties from parents.